src/array.jl

```diff
@@ -546,7 +546,65 @@ Adapt.adapt_storage(::CuArrayAdaptor{B}, xs::AbstractArray{T,N}) where {T<:Compl
 Adapt.adapt_storage(::CuArrayAdaptor{B}, xs::AbstractArray{T,N}) where {T<:Union{Float16,BFloat16},N,B} =
   isbits(xs) ? xs : CuArray{T,N,B}(xs)
 
+"""
+    cu(A::AbstractArray{T}; unified=false)
```

Comment:
This signature doesn't quite match the description. Maybe it should be two docstrings, one for arrays & one to explain what adapt? Or perhaps just more clearly organised.